crates/edr_evm/src/miner.rs

```diff
@@ -394,8 +400,14 @@ where
         }
     }
 
-    let mut block_builder =
-        ChainSpecT::BlockBuilder::new_block_builder(blockchain, state, cfg.clone(), options)?;
+    // TODO: Add support for withdrawals
```

Comment:
Could you link an issue please?

Reply:
Thanks for catching that. I added an issue and referenced it: https://github.com/NomicFoundation/edr/pull/977/commits/39c8d0968e298cbbd115865da5249384292321cf